src/v/archival/ntp_archiver_service.cc

```diff
@@ -75,7 +60,8 @@ ntp_archiver::ntp_archiver(
   , _gate()
   , _initial_backoff(conf.initial_backoff)
   , _segment_upload_timeout(conf.segment_upload_timeout)
-  , _manifest_upload_timeout(conf.manifest_upload_timeout) {
+  , _manifest_upload_timeout(conf.manifest_upload_timeout)
+  , _io_priority(conf.upload_io_priority) {
```

Comment:
it seems to me that `_io_priority` is not being used

Reply:
It's actually used on line 119. But while checking this I found that in service.cc similar field is not used and deleted it.